app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -1,6 +1,11 @@
 module ManageIQ::Providers::EmbeddedAnsible::CrudCommon
   extend ActiveSupport::Concern
 
+  def self.included(model)
+    model.include SupportsFeatureMixin
+    model.supports :create
```

Comment:
Do each of these actually support create?  The mixin doesn't really define it it just raises an exception in raw_create_in_provider so it is possible that a class that includes this doesn't define its own `raw_create_in_provider` but it'll look like it does support create

Reply:
Both of the mixees define it.

```
$ git grep CrudCommon
app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb:16:  include ManageIQ::Providers::EmbeddedAnsible::CrudCommon
app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb:17:  include ManageIQ::Providers::EmbeddedAnsible::CrudCommon
app/models/manageiq/providers/embedded_ansible/crud_common.rb:1:module ManageIQ::Providers::EmbeddedAnsible::CrudCommon

$ git grep raw_create_in_provider
app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb:26:  def self.raw_create_in_provider(manager, params)
app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb:23:  def self.raw_create_in_provider(manager, params)
app/models/manageiq/providers/embedded_ansible/crud_common.rb:37:    def raw_create_in_provider(_manager, _params)
```

The abstract method is there to force the author to create one if they mix it in, so I think it's ok to say `supports :create` here.

Reply:
yea. I had that justification but I still feel this is the wrong place